modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PolicyDetails/policy-navigator-details/index.tsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @liferay/empty-line-between-elements */
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import './index.scss';
+
+import {useEffect, useState} from 'react';
+
+import NavigationBar from '../../../../../common/components/navigation-bar';
+import sortedByDate from '../../../../../common/utils/sortedByDate';
+import DriverInfo from './driver-info';
+import HistoryInfo from './history-info';
+import arrayOfHistory from './policyNavigatorDataHistory';
+import {ApplicationPolicyDetailsType, dataJSONType} from './types';
+import VehicleInfo from './vehicle-info';
+
+enum NavBarLabel {
+	Drivers = 'Drivers',
+	Vehicles = 'Vehicles',
+	History = 'History',
+}
+
+const PolicyDetail = ({
+	dataJSON,
+	email,
+	phone,
+}: ApplicationPolicyDetailsType) => {
+	const navbarLabel = [
+		NavBarLabel.Vehicles,
+		NavBarLabel.Drivers,
+		NavBarLabel.History,
+	];
+	const [active, setActive] = useState(navbarLabel[0]);
+	const [applicationData, setApplicationData] = useState<dataJSONType>();
+	const [showPanel, setShowPanel] = useState<boolean[]>([]);
+
+	useEffect(() => {
+		try {
+			const newDataJSON = JSON.parse(dataJSON);
+			setApplicationData(newDataJSON);
+		}
+		catch (error) {
+			console.warn(error);
+		}
+	}, [dataJSON, email, phone]);
+
+	const arraySortedByDate = arrayOfHistory.sort(sortedByDate);
```

Comment:
These variables names are not clear enough

* arraySortedByDate (array of what?)
* arrayOfHistory (array and history of???)
* sortedByDate (what is sortedByDate???)

See: https://github.com/ryanmcdermott/clean-code-javascript#use-meaningful-and-pronounceable-variable-names

First add the context to the variable, then the behavior, for example:

```suggestion
	const carsSortedByDate = carsHistory.sort(carsSortedByDate);
```

I added as example `cars` 

But still there's the conflict name because `carsSortedByDate` is used twice.

Can you find a better name for this variables? please please please

Reply:
I chaged for 	`const policyHistorySortedByDate = policyHistory.sort(sortedByDate);`
Is better?

Reply:
Yes, much better

It's possible to rename `sortedByDate` to something else?

Reply:
Sure,  done!